.github/workflows/push_tests.yml

```diff
@@ -153,61 +151,6 @@ jobs:
         name: torch_cuda_test_reports
         path: reports
 
-  peft_cuda_tests:
```

Comment:
Not needed as LoRA Tests require PEFT. We can just run the LoRA tests.

Reply:
The LoRA tests are basically PEFT tests, no?

Reply:
Yup